src/app/content/PrivateHeader.js

```diff
@@ -2,45 +2,84 @@ import React from 'react';
 import { c } from 'ttag';
 import PropTypes from 'prop-types';
 import {
+    Hamburger,
     MainLogo,
     Searchbox,
-    SupportDropdown,
     UpgradeButton,
     useUser,
+    useActiveBreakpoint,
+    SearchDropdown,
     TopNavbar,
-    TopNavbarLink
+    TopNavbarLink,
+    Icon,
+    FloatingButton,
+    useModals
 } from 'react-components';
+import { isMobile as isItMobile } from 'proton-shared/lib/helpers/responsive';
 import { withRouter } from 'react-router';
 
-const PrivateHeader = ({ search, onSearch, location }) => {
+import ContactModal from '../components/ContactModal';
+
+const PrivateHeader = ({ title, search, onSearch, location, expanded, onToggleExpand }) => {
     const [{ hasPaidMail }] = useUser();
+    const { createModal } = useModals();
     const inSettings = location.pathname.startsWith('/contacts/settings');
+    const activeBreakpoint = useActiveBreakpoint();
+    const isMobile = isItMobile(activeBreakpoint);
+
     return (
         <header className="header flex flex-nowrap reset4print">
-            <MainLogo url="/contacts" />
-            {!inSettings && <Searchbox placeholder={c('Placeholder').t`Search`} value={search} onChange={onSearch} />}
+            <MainLogo url="/contacts" className="nomobile" />
+            <Hamburger expanded={expanded} onToggle={onToggleExpand} />
+            {title && isMobile ? <span className="big ellipsis">{title}</span> : null}
+            {inSettings || isMobile ? null : (
+                <Searchbox placeholder={c('Placeholder').t`Search`} value={search} onChange={onSearch} />
+            )}
             <TopNavbar>
```

Comment:
I think we need `<TopNavbar className="mr1">`. Otherwise there's no separation between the navigation bar and the add button

Reply:
As you want, but please tell me if there are updates on it in order to harmonize with Angular ;)